### DISCUSSION TITLE
articles/daddy-is-smartest

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#126] [Comments]
https://hutusi.com/articles/daddy-is-smartest 

 晚上我从网上的视频学了个小魔术变给大卫看，大卫非常开心，并且惊奇的跟妈妈说：“爸爸太聪明了，他会变魔术!”

**Comment by zjiekai** [comment]
比抢东西的猴子还要笨！！ 哈哈哈哈

**Comment by ColarXu** [comment]
我也是一惊，马上去确认了一下 pointless 的意思。想问问你小孩，为什么 life is pointless, 以及他心目中的聪明指的是什么。哈哈，好奇小孩的想法。

**Comment by hutusi** [comment]
> 我也是一惊，马上去确认了一下 pointless 的意思。想问问你小孩，为什么 life is pointless, 以及他心目中的聪明指的是什么。哈哈，好奇小孩的想法。

@ColarXu 他也不清楚"life is pointless"的真正含义，跟我说是在《dogman》书上看到的。至于聪明，就是知道很多东西。

**Comment by hutusi** [comment]
@zjiekai 哈哈，童言无忌~

**Comment by tumars** [comment]
为啥小朋友的英语这么好啊==，你们生活在国外？

**Comment by hutusi** [comment]
@tumars 生活在国内，不过从小开始英语学习~

**Comment by petite-dd** [comment]
哈哈，优秀了优秀了，看儿子疼老爹，老妈是吃醋了

**Comment by hutusi** [comment]
@petite-dd 小朋友的世界比较单纯😀
